.changeset/little-pens-change.md

```diff
@@ -0,0 +1,5 @@
+---
+'@shopify/polaris-tokens': minor
+---
+
+Expose token group types
```

Comment:
Maybe also include that we are exporting the scale and/or alias for certain token groups as well. Just for improved changelog searching 🔍